tests/unit/data/test_particle_collection.py

```diff
@@ -40,14 +42,15 @@ def test_exceptions(particle_database):
     new_particle = create_particle(
         template_particle=particle_database["gamma"], name="gamma_new"
     )
-    particle_database += new_particle
+    dummy_database = deepcopy(particle_database)
+    dummy_database += new_particle
```

Comment:
Hmm I thought using the `scope="class"` for this fixture was the preferred solution?

Reply:
Ok, testing whether it works as expected

Reply:
Nope, also using
```python
@pytest.fixture(scope="class")
def particle_database() -> ParticleCollection:
    ...
```
instead of
https://github.com/ComPWA/expertsystem/blob/21d0cf79a4bec0f88ea5b69f3a5ead79c9187e11/tests/unit/conftest.py#L20-L23

also kills the channel tests. So seems that the `deepcopy` is the safest solution, even if dirty. Something to keep in mind for #219.

Reply:
Ah, I get it now. Even thought the fixture will be executed and destroyed many times (on the class level scope), the  DATABASE is a global still persists the old state.
So in order to really have fixture we need to get rid of this DATABASE global. In this case it would be enough to just load a `ParticleCollection` from the pdg or yaml file and just return that.

Reply:
71fd1fb73da35186b2bc0034b6fad24342b3ab1b would have been easier... :man_facepalming: